app/Mail/ContactFormEmail.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+use App\ContactForm;
+
+class ContactFormEmail extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    /**
+     * Create a new message instance.
+     *
+     * @param ContactForm $contactForm
+     */
+    public function __construct(ContactForm $contactForm)
+    {
+        $this->contactForm = $contactForm;
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        return $this->view('mail.contact')
+                    ->with(['contactForm' => $this->contactForm]);
```

Comment:
PHPStorm complains: Property `contactForm` not found in ContactFormEmail

Reply:
that's actually something I forgot to declare as a property for the class but it still works.. I'll add it anyway in a new commit in a sec.

Reply:
error should be gone now